vizro-core/src/vizro/models/_navigation/nav_link.py

```diff
@@ -62,7 +63,7 @@ def build(self, *, active_page_id=None):
                 f"Page with ID {first_page_id} cannot be found. Please add the page to `Dashboard.pages`"
             ) from exc
 
-        button = dbc.Button(
+        nav_link = dbc.NavLink(
```

Comment:
My decision to call this `vm.NavLink` feels vindicated now this actually returns a `dbc.NavLink` 😅